go-controller/pkg/node/gateway_localnet.go

```diff
@@ -221,8 +221,12 @@ func (npw *localPortWatcherData) addService(svc *kapi.Service) error {
 			}
 
 			if gatewayIP != "" {
+				// Fix Azure/GCP LoadBalancers. They will forward traffic directly to the node with the
+				// dest address as the load-balancer ingress IP and port
+				iptRules = append(iptRules, getLoadBalancerIPTRules(svc, port, gatewayIP, port.NodePort)...)
```

Comment:
can we do that in follow up (if even necessary since we are going away from local gw)? We never had that in our donwstream OCP hacks unless I'm missing something?

Reply:
`syncServices` is new in the last week... and yes, I think I failed to update the OCP-local hack correctly.

Reply:
or it could have been my fault on the rebase, either way let's follow up on it later